tools/docker/firecracker-runtime.json

```diff
@@ -0,0 +1,13 @@
+{
+  "firecracker_binary_path": "/usr/local/bin/firecracker",
+  "socket_path": "./firecracker.sock",
```

Comment:
If you rebase, you can now remove the `socket_path` field.

Reply:
Nice, updating